pyfr/plugins/fluidforce.py

```diff
@@ -21,7 +21,10 @@ def __init__(self, intg, cfgsect, suffix):
         self.nsteps = self.cfg.getint(cfgsect, 'nsteps')
 
         # Check if we need to compute viscous force
-        self._viscous = intg.system.name == 'navier-stokes'
+        self._viscous = intg.system.name == ('navier-stokes' or 'ac-navier-stokes')
```

Comment:
This wont work.  `('navier-stokes' or 'ac-navier-stokes')` evaluates to `'navier-stokes'`.  The fluid force plugin is already messy enough with how it handles the viscous terms without yet another bunch of if statements to handle AC.  Rework.